libtcmu.c

```diff
@@ -432,8 +434,8 @@ static int add_device(struct tcmulib_context *ctx, char *dev_name,
 
 	dev->handler = find_handler(ctx, dev->cfgstring);
 	if (!dev->handler) {
-		tcmu_err("could not find handler for %s\n", dev->dev_name);
-		goto err_free;
+		tcmu_dbg("could not find handler for %s\n", dev->dev_name);
```

Comment:
should this be a warning tcmu_warn () ?
For tcmu-runner case, it is helpful to have this msg at logs with non-debug log levels too.

Reply:
Yeah, that makes sense to me, thanks.